osquery/tables/system/hash.cpp

```diff
@@ -182,6 +186,21 @@ bool FileHashCache::load(const std::string& path, MultiHashes& out) {
   return true;
 }
 
+std::string genSsdeepForFile(const std::string& path) {
+#ifndef WINDOWS
```

Comment:
May be it is better to use `#ifdef OSQUERY_POSIX` ? It is not a big deal until `not WINDOWS == OSQUERY_POSIX`. But it may be changed one day.